src/viztracer/modules/vcompressor/vcompressor.c

```diff
@@ -184,6 +188,14 @@ static PyObject* vcompressor_compress(VcompressorObject* self, PyObject* args)
 
     dump_parsed_trace_events(parsed_events, fptr);
 
+    // file_info here should not be released
```

Comment:
Okay this is a borrowed reference. Could you add this (more explicitly than should not be released) to the comment? Just say this is a borrowed reference would be fine.